### DISCUSSION TITLE
Who wants to take ownership of the PhotoView product?

### DISCUSSION THREAD

**Discussion by kkovaletp** [photoview/photoview#893] [General]
According to the frequency of commits and amount of pending PRs, it looks like the current repo owner @viktorstrate doesn't maintain it anymore.

But as new issues are reported, PRs created, and even sometimes new discussions are started, there is an active community of product users, interested in product growth and bug fixes.

So, I'd like to ask whether we have here a developer (or a group of developers), familiar with the technology stack and has enough experience to take ownership of the product and maintain it.

From my point of view, the requirements for this role are:
- The person, who commits to be a new product owner, should be ready to spend the required time at least once a week to answer new questions, review reported issues and created PRs, contribute to the product, and publish new releases and hotfixes.
- To avoid the current "no active maintainer" case, the new owner needs to add at least 1 (better several) backup maintainers to the repo with appropriate permissions. If there will be a group of owners, this requirement is satisfied, however, it is still better to have more than 1 maintainer per area (frontend, backend, infrastructure).
- This repo already has a list of open PRs, issues, and discussions, which are waiting for review. A new owner has to migrate all of them to the new project, review, and act accordingly to the review results.

Technically I see the process as:
- Fork this project's repo to a new space
- Setup CI\CD and Docker infrastructure
- Copy all the open PRs and issues to the new repo "as is" and add a link to new ones in this repo as a comment
- Probably rename the product to something like "Libre PhotoView" - could discuss later
- Do the review of all the pending content in the new repo, answer discussions here
- Make the 1st release, based on accepted PRs, bug fixes, and potentially some new features

Waiting for your opinions in the comments)

**Comment by kkovaletp** [comment]
I could help with the Infrastructure area maintenance in the new repo, but I'm not a developer, so it doesn't make sense to become a repo owner from my side - that is why I am asking here for volunteers with relevant development experience, willing to step in

**Comment by ddshd** [comment]
Just FYI, message from maintainer:

> thank you for your interest in the project. I have just started working on my masters thesis and I don’t have much time to work on Photoview at the time.. I have certainly not abandoned the project, but it might take some time before I will have the time to properly update it and look through all the PRs.

https://discord.com/channels/794576839813234688/794576839813234696/1149785865049079840

**Reply by kkovaletp** [reply]
Thanks! For some reason, I didn't notice there is a Discord for the project)
I've added a comment about this discussion in that channel, so that the owner of the repo and others, interested in the product development, might notice it with a higher probability.

As I mentioned there, I completely understand all the personal reasons, but to handle that it is a good practice to have a team of maintainers, so that if someone has to take a break, others could substitute his/her. If such maintainer(s) will be found and set for this repo, for me, it would be the preferable solution, as no migration is needed.

**Comment by jult** [comment]


**Reply by kkovaletp** [reply]
there is no any estimation from the owner so far

**Comment by kkovaletp** [comment]
As a next step in this discussion, I've created a PR #910 with the most interesting and ready-to-be-merged fixes and improvements for the product.
I'll keep merging there new fixes and improvements once new PRs published in the project repo.
This PR is going to be an alternative to the project's master branch while the project is on hold.

So, feel free to check that PR and use it. The PR's description contains a how-to guide to deploy the app from the source code.

BTW, while I cover outdated base image and 3rd-party software issue by rebuilding the image locally, we still have outdated dependencies in the source code, so if we have some skilled backend and frontent developers, who would like to contribute and help the community, it will be great to see PRs with updated dependencies and migration to the latest Node LTS version

**Comment by kkovaletp** [comment]
The project restarted with 2 active maintainers, so closing this discussion

**Comment by googollee** [comment]
Hello. Do you need more maintainers? I want to help on the backend side, especially the SQL query.

**Reply by kkovaletp** [reply]
Hello @googollee, sure. We have a lot of tasks for the Backend and Database) 

We'd appreciate your help)
Would you be able to commit to regularly spending at least some minimal time on the project to perform code reviews, consultancy on development activities, and contributions to address existing open issues?
Can you please share your development profile (LinkedIn page, CV, or anything similar, where we could see your experience and expertise)? You can send it to me privately in Discord direct messages if you don't want to share it publicly. Then I'll forward it to our internal maintainers chat so that @viktorstrate and @jordy2254 can analyze it as well.

@viktorstrate, @jordy2254, what do you think?

**Reply by googollee** [reply]
You can find my profile on https://www.linkedin.com/in/googollee/.

**Reply by googollee** [reply]
Also, find me on Discord by `googollee`. What's your Discord id?

**Reply by kkovaletp** [reply]
Impressive profile!
My Discord ID is `kkb.graph` but I cannot find you in Discord. Are you a member of the Community server? The link is in the Readme: https://discord.gg/jQ392948u9
